### DISCUSSION TITLE
Look of components in Storybook

### DISCUSSION THREAD

**Discussion by UniversalTraveller** [neuefische/questions#131] [Web] [answered]
Hello,

when I put my components in Storybook, they have margins and padding I don't get in the app. The layout looks wrong. I guess that's because I use a CSS reset in my `styles.js`.

For example this is my tagging component as it looks in the app:
<img width="682" alt="Screenshot 2022-08-31 at 21 33 57" src="https://user-images.githubusercontent.com/59486612/187766798-46f31a78-3604-42ec-b9d7-351dc7be2d58.png">

And this is the same in Storybook:
<img width="654" alt="Screenshot 2022-08-31 at 21 33 32" src="https://user-images.githubusercontent.com/59486612/187766808-0d2b4c50-0440-4d00-9c25-d951b87dd122.png">

What should I do differently?

**Comment by alexclaes** [comment, marked as answer]
Hi,

yes - that's most probably the reason!

You need to make sure, that Storybook knows about your global styles (such as the CSS reset). 

The file `.storybook/preview.js` allows you to configure this.
https://storybook.js.org/docs/react/configure/overview#configure-story-rendering

With the help of `decorators` you can add global styles to all rendered stories. Here is an example with Styled Components:
https://github.com/aclaes/nf-quiz-app-react/blob/storybook/bo-web-22-1/.storybook/preview.js

**Reply by UniversalTraveller** [reply]
Thank you, Alex! That's the solution, works like a charm.
